orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/OperationsController.groovy

```diff
@@ -472,12 +467,28 @@ class OperationsController {
       config = preprocessor.process(config)
     }
 
-    def json = objectMapper.writeValueAsString(config)
-    log.info('requested task:{}', json)
-    def pipeline = executionLauncher.start(ORCHESTRATION, json)
+    def pipeline = null
+    try {
+      pipeline = executionLauncher.start(ORCHESTRATION, json)
+    } finally {
+      log.info('started execution {} from requested task: {}', pipeline?.id, renderForLogs(config))
+    }
     [ref: "/tasks/${pipeline.id}".toString()]
   }
 
+  private String renderForLogs(Map config) {
+    if (!log.isInfoEnabled()) {
+      return
+    }
+
+    def json = objectMapper.writeValueAsString(config)
+    if (json.length() < 1_000_000) {
+      return json
+    }
+
+    return json[0..50_000] + " (...) " + json[-50_000..-1]
```

Comment:
The only Issue I see with this approach is it will make it so it can't be parsed. Maybe some indication that it was truncated not 50,000 characters in would be nice.

Reply:
Good point, I'll tune the log message. Although lifting a multi-MB json from the logs isn't exactly practical either, I think it's more important to log the execution id so we can retrieve the execution later

Reply:
I'm assuming we don't get a nice `Content-Length` header shipped across? That would be too convenient eh?

Reply:
@ajordens that reminds me, the gate request logger reports `responseSize` but not `requestSize`, that would be nice to have for this use case!

Reply:
added length and fixed silly mistake in bd350448a